Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN mkdir /helm-go-lib-build && \
     cd /helm-go-lib-build && \
     git clone https://github.com/broadinstitute/helm-scala-sdk.git && \
     cd helm-scala-sdk && \
-    git checkout master && \
+    git checkout rt-uninstall && \
```

Comment:
TODO change back to master once https://github.com/broadinstitute/helm-scala-sdk/pull/5 is merged